libs/SalesforceHybrid/build.gradle.kts

```diff
@@ -78,4 +78,8 @@ android {
         renderScript = true
         aidl = true
     }
+
+    kotlin {
+        jvmToolchain(17)
```

Comment:
I found this build failure, which actually looked like a lingering regression from the Gradle 8 work on `dev` already.